### PR TITLE
DT-445 Migrate gnome-logs to Gnome42/Core22

### PR DESCRIPTION
**Pull request by sergio-costas**
# Pull Request Template

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please check only the options that are relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

**Test Configuration**:

* OS (please include version):
* Any other relevant environment information:

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

**Comment by jbicha** [comment]
Could you override the version number? This upstream's maintainer historically often doesn't update the version numbers in the AppStream metadata so by default snap (and Flatpak) report that the version is still 3.34.0.

File: snapcraft.yaml [hunk 1]
```diff
@@ -32,36 +32,37 @@ parts:
...
+      craftctl default
+      mkdir -p $CRAFT_PART_INSTALL/meta/gui/
+      cp ../src/data/icons/scalable/org.gnome.Logs.svg $CRAFT_PART_INSTALL/meta/gui/
 
   libraries:
+    after: [ gnome-logs ]
     plugin: nil
     prime:
-      - "-*"
+      - -"*"
```
Comment (hellsworth) [inline snapcraft.yaml]: So you want to unprime literally everything in the prime dir? How does this work to give us a working app?

**Comment by sergio-costas** [comment]
> Could you override the version number? This upstream's maintainer historically often doesn't update the version numbers in the AppStream metadata so by default snap (and Flatpak) report that the version is still 3.34.0.

How can I do it? I already am using `craftctl set version=$(git describe --tags --abbrev=10 | sed -e s/gnome-logs-//g)` in the `override pull` section...

File: snapcraft.yaml (hunk 1 above)
Comment (sergio-costas) [reply snapcraft.yaml]: Mmm... good point... But what is the difference between both statements?

Comment (sergio-costas) [reply snapcraft.yaml]: In fact... why is there that part? I think it does nothing...

Comment (kenvandine) [reply snapcraft.yaml]: Indeed, just drop the libraries part completely.  This is leftover from when we had been staging packages as depends.

Comment (sergio-costas) [reply snapcraft.yaml]: Fixed.

**Review by hellsworth — APPROVED** [review]
looks great thanks!

**Pull request merged**